mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -63,9 +63,12 @@ public static SentMail fromMail(Mail mail ) throws MessagingException {
             .message(mail.getMessage())
             .state(mail.getState())
             .attributes(buildAttributesMap(mail))
+            .attribute(SMTP_AUTH_USER_ATTRIBUTE_NAME, "true")
```

Comment:
you probably want to have a `fromMailet()` method of `sentMailBuilder` instead ?